clone-cert.sh

```diff
@@ -67,14 +68,20 @@ KEY_LEN="$(openssl x509  -in "$ORIG_CERT_FILE" -noout -text \
 MY_PRIV_KEY="$DIR$HOST.$KEY_LEN.key"
 MY_PUBL_KEY="$DIR$HOST.$KEY_LEN.cert"
 
-offset="$(openssl asn1parse -in "$ORIG_CERT_FILE" | grep SEQUENCE \
-    | tail -n1 |sed 's/ \+\([0-9]\+\):.*/\1/' | head -n1)"
+if [ "$OS" == "Darwin" ];
+then
+    offset="$(openssl asn1parse -in "$ORIG_CERT_FILE" | grep SEQUENCE \
+        | tail -n1 | head -n1 | awk '{print $1}' | sed 's/:.*//')"
```

Comment:
If this line does in Darwin what it is supposed to do, shouldn't it work in bash as well? If so, we wouldn't need the conditional.

A quick test on my system showed that it does indeed do the same. So I suggest to simply replace the original line with this one if the original line does not work on Darwin.

Reply:
Sure, didn't have time to test in on Linux, and would like to preserve your code over mine if possible.
I'm ok to keep only the new one if it works on linux
Also here, I think we should add awk to the deps_list as I'm not sure that every linux box comes with awk preinstalled